frontend/sozisel-app/src/App.tsx

```diff
@@ -36,6 +37,11 @@ export default function App(): ReactElement {
           <AuthRoute path="/sessions" component={SessionsList} />
           <AuthRoute path="/sessions/create" component={CreateSession} />
           <AuthRoute path="/sessions/:id/edit" component={EditSession} />
+          <AuthRoute
+            exact={false}
+            path="/session-result/:id"
```

Comment:
I think we should stick to one path naming pattern (`/sessions/:id/<action>`) in this case it will be: `/sessions/:id/result`